### ISSUE TITLE
please increment versionCode with each release

### ISSUE THREAD

**Issue by IzzySoft** [shazomii/Notely#32]
Your app now had 3 releases, but `versionCode` still is set to `1`. Some background of this variable: it is used by Android internally to tell versions apart – so a higher `versionCode` is interpreted as an update, an equal `versionCode` as the same version, and a lower one as a "downgrade". Currently, while your app [is listed in my repo](https://apt.izzysoft.de/fdroid/index/apk/com.davenet.notely), existing users never get update notifications: though the APK is "replaced" (being considered the same version), the F-Droid client on the users' devices do not consider it a newer version and thus ignore it.

TL;DR: always before you make a release, ideally as the last step before tagging, please remember to increase `versionCode`. Thanks!

**Comment by shazomii** [comment]
Noted. I'll adjust accordingly.
